### PR TITLE
Devapm

### PR DESCRIPTION
**Pull request by Song2017**
add phpMyAdmin_*

### CODE REVIEW

File: docker-library/apache-php-mysql/README.md [hunk 1]
```diff
@@ -60,3 +59,4 @@ The startup log file (**entrypoint.log**) is placed under the folder /home/LogFi
 ## Change Log
 - **Version 0.2** 
   1. Supports uploading large files. See [php.ini](0.2/php.ini) here.
+  2. Add App Setting Key: DATABASE_TYPE. See [entrypoint.sh](0.2/entrypoint.sh),[azuredeploy.json](azuredeploy.json) here.
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/README.md]: add more description for this to make users understand this app setting item.

File: docker-library/apache-php-mysql/azuredeploy.json [hunk 2]
```diff
@@ -51,19 +43,10 @@
             "type": "Microsoft.Web/sites",
             "name": "[parameters('siteName')]",
             "properties": {
-                "siteConfig": {
-                    "appSettings": [
+                "appSettings": [
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/azuredeploy.json]: is this "appSettings" parallel now to "siteConfig" ?

File: docker-library/apache-php-mysql/README.md [hunk 3]
```diff
@@ -17,21 +17,20 @@ Ubuntu 16.04 is used as the base image.
 This docker image enables you to:
 
 - run a Apache/PHP/MySQL Environment on **Azure Web App on Linux**;
-- connect your App site to **Azure ClearDB** or the builtin MariaDB;
+- set DATABASE_TYPE with **local** will run built-in MariaDB and phpMyAdmin;
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/README.md]: merge this into "manage the built-in MariaDB with the built-in phpMyAdmin"

File: docker-library/apache-php-mysql/README.md [hunk 4]
```diff
@@ -17,21 +17,20 @@ Ubuntu 16.04 is used as the base image.
...
+- connect your App site to **Azure ClearDB** or the built-in MariaDB;
 - manage the build-in MariaDB with the built-in phpMyAdmin;
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/README.md]: merge the second in to this line to make it looking like below:

manage the build-in MariaDB with the built-in phpMyAdmin (You need set DATABASE_TYPE to "LOCAL")

File: docker-library/apache-php-mysql/0.2/entrypoint.sh [hunk 5]
```diff
@@ -72,49 +70,46 @@ service ssh start
...
-	echo "INFO: $PHPMYADMIN_HOME/config.inc.php already exists."
-fi
+update_settings
 
-echo "Loading phpMyAdmin conf ..."
-if ! grep -q "^Include conf/httpd-phpmyadmin.conf" $HTTPD_CONF_FILE; then
-	echo 'Include conf/httpd-phpmyadmin.conf' >> $HTTPD_CONF_FILE
+if [ "${DATABASE_TYPE,,}" = "local" ]; then
+
+	echo "INFO: Local Mariadb and phpMyAdmin Loading ..."
```
Comment (fanjeffrey) [inline docker-library/apache-php-mysql/0.2/entrypoint.sh]: "INFO: loading local MariaDB and phpMyAdmin ..."

**Review by fanjeffrey — CHANGES_REQUESTED** [review]

File: docker-library/apache-php-mysql/README.md (hunk 3 above)
Comment (Song2017) [reply docker-library/apache-php-mysql/README.md]: yeah

**Review by fanjeffrey — APPROVED** [review]

**Pull request merged**